mc/gui/main_window.py

```diff
@@ -176,28 +176,25 @@ def setup_systray(self):
         )
         self.system_tray.tray_breathing_enabled_qaction.setDisabled(True)
 
-        count_int = 0
         self.system_tray.tray_phrase_qaction_list.clear()
-        for l_phrase in mc.model.PhrasesM.get_all():
-            INDENTATION_STR = "  "
-            ACTIVE_MARKER_STR = "•"
-            INACTIVE_MARKER_STR = " "
-            active_or_inactive_str = INACTIVE_MARKER_STR
-            if l_phrase.id_int == mc.mc_global.active_phrase_id_it:
-                active_or_inactive_str = ACTIVE_MARKER_STR
-            tray_phrase_qaction = QtWidgets.QAction(active_or_inactive_str + INDENTATION_STR + l_phrase.title_str)
-            tray_phrase_qaction.triggered.connect(
-                functools.partial(
-                    self.phrase_list_widget.on_new_row_selected_from_system_tray,
-                    l_phrase.id_int
+        for count in enumerate(range(5)):
```

Comment:
Can you try something like this instead?: `enumerate(mc.model.PhrasesM.get_all())`. Using this approach you don't need a double for loop (but have to keep the code with `break`)

Reply:
This would give as a result a tuple with the count in the first position and a "phrase" element in the second position. Do you know about "unpacking" in Python? This makes it very elegant like this: `for (count, element) in enumerate(mc.model.PhrasesM.get_all())`. Not sure about all the details so please test the code (by starting the application, which should be enough, but just in case please also open the systray since this code is about setting that up)

Reply:
yeah thinks, I would like to try this!!